lib/assert.js

```diff
@@ -151,6 +154,16 @@ x.ifError = (err, msg) => {
 	test(!err, create(err, 'Error', '!==', msg, x.ifError));
 };
 
+x.jsxEqual = (val, expected, msg) => {
+	const treeEquals = reactElementToJSXString(val) === reactElementToJSXString(expected);
+	test(treeEquals, create(val, expected, '===', msg, x.jsxEqual));
+};
+
+x.jsxNotEqual = (val, expected, msg) => {
```

Comment:
I agree about it looking uglier, but better to be consistent and always have `not` first. In a perfect world we would use snakecase in JS...